packages/cli/src/commands/diagnose.ts

```diff
@@ -36,12 +37,18 @@ export const diagnose = ({
     process.env["APPSIGNAL_APP_ENV"] = environment
   }
 
-  spawnSync(
-    `${cwd}/node_modules/@appsignal/nodejs/bin/diagnose`,
-    !report ? ["--no-report"] : undefined,
-    {
-      cwd,
-      stdio: "inherit"
-    }
-  )
+  let args = []
+  switch (sendReport) {
+    case true:
+      args.push("--send-report")
+      break
+    case false:
+      args.push("--send-report")
```

Comment:
Shouldn't this one set the "no send" flag?

```suggestion
      args.push("--no-send-report")
```

Reply:
It should yes :)